### PR TITLE
Center map

### PR DESCRIPTION
**Pull request by prachigarg19**

### CODE REVIEW

**Comment by mariahosfeld** [comment]
Tested the functionality, works for me as expected.

File: src/features/projects/components/ProjectsMap.tsx [hunk 1]
```diff
@@ -161,6 +161,17 @@ export default function ProjectsMap(): ReactElement {
...
+      const newViewport = {
+        ...viewport,
+        latitude: 36.96,
+        longitude: 0,
+      };
+      setViewPort(newViewport);
+    }
+  }, [showProjectList, router]);
+
   return (
```
Comment (mohitb35) [inline src/features/projects/components/ProjectsMap.tsx:175]: @prachigarg19 
1. Not clear why `router` is a dependency here.
2. Please check the combination of `embed` and `showProjectList` as `showProjectList` is not supposed to affect anything without `embed = true`

**Review by mohitb35 — CHANGES_REQUESTED** [review]
@prachigarg19 I've taken a look and suggested some minor changes.

Also, I think the new viewport gets lost if the user goes to a single project page and comes back (the map no longer remains centered in this case)

File: src/features/projects/components/ProjectsMap.tsx (hunk 1 above)
Comment (prachigarg19) [reply src/features/projects/components/ProjectsMap.tsx:175]: ` router` works for initial render.

**Comment by mariahosfeld** [comment]
I checked again, the project_list=false context is lost when going back from a single project view.

**Comment by mariahosfeld** [comment]
Functioniality now works for me.

**Review by norbertschuler — APPROVED** [review]
This works, but make the map even more Europe centered as before as the map always get reseted to these coordinates when hiding or showing projects even if you currently looking at another spot of the world.

**Pull request merged**